src/book/02-system/01-foundations/05-mathematics.mdx

```diff
@@ -315,3 +317,34 @@ For measurements on the field we use field space. For field coordinates the **x-
 - [Wolfram MathWorld](http://mathworld.wolfram.com/)
 
 - [Eigen Documentation](http://eigen.tuxfamily.org/dox/)
+
+# Interpolation
+
+Given a set of data points, often we want to find a function that contains all these points, which we call an interpolant. Interpolants are commonly polynomials, because they can approximate complex curves with a few points, with relatively low computational costs. However, interpolant polynomials which are locally accurate for the given data can have oscillation issues at the edges of the interval which is being interpolated on - [Runge's phenomenon](https://en.wikipedia.org/wiki/Runge%27s_phenomenon).
```

Comment:
We can mention it. I assumed enough contraints would be included to make the interpolant unique, but I guess that should be explicily stated.

Reply:
Aliasing can also happen when you have a very large degree on your polynomial. You can get extra wiggles occurring between interpolation points that may not be intended even though the end points align with the the next polynomial in the spline. I think splines are generally broken up in a way to avoid this anyway, but might be worth highlighting this (feel free to override me here if you think its unnecessary)

Reply:
Yeah I guess if you use a polynomial that has degree higher than you need. I've added in some stuff, let me know if it's what you were going for.